homie/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "homie"
+version = "0.1.0"
+authors = ["Andrew Walbran <qwandor@google.com>", "David Laban <alsuren@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+async-channel = "1.4.0"
+futures = "*"
+local_ipaddress = "0.1"
+log = "*"
```

Comment:
(optional) Probably worth pinning these versions down if we ever publish on crates.io. For now we can rely on Cargo.lock to pin things for us, but this won't work when we publish, I don't think.

Reply:
Done in #19.